entries/r/ripio.com.json

```
@@ -0,0 +1,15 @@
+{
+  "Ripio": {
+    "domain": "ripio.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://launchpad.ripio.com/blog/que-es-autenticacion-dos-factores-2fa",
+    "categories": [
+      "cryptocurrencies"
+    ],
+    "regions": [
+      "br"
```

Also operates in Argentina.
```suggestion
      "ar",
      "br"
```